oars/models/enrollment/enrollments_by_day.sql

```diff
@@ -1,16 +1,27 @@
--- one record per learner per day with enrollment status
+-- one record per learner, org, course, and day with
+-- enrollment status as of 11:59 UTC of that day
+--
+-- a range of dates is calculated by converting DateTime fields
+-- to Unix timestamps, then generating a range of Unix timestamps,
+-- creating one record per timestamp (via arrayJoin), and then converting
+-- the timestamps back into dates
+
 
 select
-  d.date_day,
-  e.actor_id,
-  e.object_id,
-  e.course_id,
-  e.org,
-  e.event_type as enrollment_status,
-  e.enrollment_mode
+  date(fromUnixTimestamp(
+    arrayJoin(
+      range(
+        toUnixTimestamp(window_start),
+        toUnixTimestamp(window_end),
+        24 * 60 * 60
+      )
+    )
+  )) as the_date,
```

Comment:
Maybe `enrollment_status_at` (dbt's preferred style for dates) or `enrollment_status_date`?